app/models/manageiq/providers/kubernetes/container_manager/refresh_parser.rb

```diff
@@ -337,6 +337,7 @@ def parse_container_definition(container_def, pod_id)
         :name              => container_def.name,
         :image             => container_def.image,
         :image_pull_policy => container_def.imagePullPolicy,
+        :command           => container_def.command ? container_def.command.to_json : nil,
```

Comment:
@simon3z 
from types.go on command:
"... The command is simply exec'd, it is not run inside a shell, so traditional shell instructions ('|', etc) won't work."
from shellwords:
"This module manipulates strings according to the word parsing rules of the UNIX Bourne shell."